src/fastga/models/weight/cg/cg_components/a_airframe/a5_landing_gear_cg.py

```diff
@@ -41,7 +41,16 @@ def setup(self):
         self.add_input("data:geometry:wing:MAC:length", val=np.nan, units="m")
         self.add_input("data:geometry:wing:MAC:at25percent:x", val=np.nan, units="m")
         self.add_input("data:weight:aircraft:CG:aft:MAC_position", val=np.nan)
-        self.add_input("settings:weight:airframe:landing_gear:front:weight_ratio", val=0.3)
+        self.add_input(
+            "settings:weight:airframe:landing_gear:front:weight_ratio",
+            val=0.3,
+            desc="Should not be lower than 0.08. Roskam",
```

Comment:
If possible, I'd rather add that to the existing description in models/variable_descriptions.txt

Reply:
Done in [commit](https://github.com/supaero-aircraft-design/FAST-GA/pull/187/commits/fba594d37daaff6252539d78549a553fe8e70911)